Presentation.Web/Controllers/OData/BaseEntityController.cs

```diff
@@ -185,5 +200,47 @@ public virtual IHttpActionResult Delete(int key)
 
             return StatusCode(HttpStatusCode.NoContent);
         }
+
+        protected bool AllowOrganizationAccess(int organizationId)
```

Comment:
Samme kommentar omkring Strategy som i andet hierarki

Reply:
Jo det skal være to strategies. Det giver mening.